tests/test_conf.py

```diff
@@ -246,6 +260,47 @@ def test_param_retrieval(self, config, kwargs, cli_args, expected):
                     assert getattr(conf, fn)() == value
 
     @pytest.mark.parametrize(('config', 'expected'), [
+        ({'default': {'token_secrets': 'secret'}},
+         {'secret': None}),
+
+        ({'default': {'token_secrets': 'secret:'}},
+         Exception),
+
+        ({'default': {'token_secrets': 'secret:/'}},
+         Exception),
+
+        ({'default': {'token_secrets': 'secret:path'}},
+         {'secret': 'path'}),
+
+        ({'default': {'token_secrets': 'secret:path:with:colons'}},
+         {'secret': 'path:with:colons'}),
+
+        ({'default': {'token_secrets': 'secret:path secret2:path2'}},
+         {'secret': 'path', 'secret2': 'path2'}),
+
+        ({'default': {'token_secrets': 'secret:path secret2:path2 secret3:path3'}},
+         {'secret': 'path', 'secret2': 'path2', 'secret3': 'path3'}),
+
+        ({'default': {'token_secrets': 'secret:path secret2 secret3:path3'}},
+         {'secret': 'path', 'secret2': None, 'secret3': 'path3'}),
+
+        ({'default': {'token_secrets': '\n   secret:path     secret2\n\n secret3:path3'}},
+         {'secret': 'path', 'secret2': None, 'secret3': 'path3'}),
+
+        ({'default': {'token_secrets': '\t\n   secret:path   \t\t  secret2\n\n \tsecret3:path3 \n\t\n'}},
+         {'secret': 'path', 'secret2': None, 'secret3': 'path3'}),
+    ])
+    def test_get_token_secrets(self, config, expected):
+        with self.config_file(config) as config_file:
+            conf = Configuration(conf_file=config_file)
+
+            if expected is not Exception:
```

Comment:
We should be expecting `OsbsValidationException` here - this test would make sure no unclear exceptions are thrown